### PR TITLE
Fix 2234

### PR DESCRIPTION
**Pull request by ddcat1115**
First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!
- [ ] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
- [ ] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
- [ ] Rebase before creating a PR to keep commit history clear.
- [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: components/cascader/style/index.less [hunk 1]
```diff
@@ -9,14 +9,14 @@
     display: block;
     cursor: pointer;
     width: 100%;
+    background: rgba(255, 255, 255, 0);
+    z-index: 1;
```
Comment (afc163) [inline components/cascader/style/index.less]: 这个 `z-index` 不好吧，会超出别的浮层。

Comment (ddcat1115) [inline components/cascader/style/index.less]: 看到checkbox也是 1，一般的浮层好像高得多吧

主要这里的样式是利用input的，文案又是在同级的另外一个元素上，hover到文案上就没样式了，所以目前的处理是，label能看不能hover/click

Comment (afc163) [inline components/cascader/style/index.less]: 👌

**Comment by ddcat1115** [comment]
分支选错了，重新pr

**Pull request closed without merging**